card/mod/account/spec/set/all/account_spec.rb

```diff
@@ -1,139 +1,132 @@
 # -*- encoding : utf-8 -*-
 
-describe Card::Set::All::Account do
+RSpec.describe Card::Set::All::Account do
   describe "accountable?" do
     it "is false for cards with *accountable rule off" do
       expect(Card["A"].accountable?).to eq(false)
     end
 
-    it "is true for cards with *accountable rule on" do
-      Card::Auth.as_bot do
-        Card.create name: "A+*self+*accountable", content: "1"
-        Card.create name: "*account+*right+*create",
-                    content: "[[Anyone Signed In]]"
-      end
+    it "is true for cards with *accountable rule on", as_bot: true do
+      Card.create name: "A+*self+*accountable", content: "1"
+      Card.create name: "*account+*right+*create",
+                  content: "[[Anyone Signed In]]"
       expect(Card["A"].accountable?).to eq(true)
     end
-  end
-
-  describe "parties" do
-    it "for Wagn Bot" do
-      Card::Auth.current_id = Card::WagnBotID
-      expect(Card::Auth.current.parties.sort).to eq(
-        [Card::WagnBotID, Card::AnyoneSignedInID, Card::AdministratorID]
-      )
-    end
 
-    it "for Anonymous" do
-      Card::Auth.current_id = Card::AnonymousID
-      expect(Card::Auth.current.parties.sort).to eq([Card::AnonymousID])
-    end
-
-    context "for Joe User" do
-      before do
-        @joe_user_card = Card::Auth.current
-        @parties = @joe_user_card.parties # note: must be called to test resets
+    describe "parties" do
+      it "for Wagn Bot", with_user: Card::WagnBotID do
+        expect(Card::Auth.current.parties.sort).to eq(
+                                                     [Card::WagnBotID, Card::AnyoneSignedInID, Card::AdministratorID]
```

Comment:
not clear on our new line length policy, but this wraps on me at github with the browser at full screen (in split diff mode).